tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,209 @@
+		}
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that a PipelineRun is created in dev namespace.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, componentName)
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that in managed namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, "release")
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies a PipelineRun started in managed namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("tests a Release should have been created in the dev namespace and succeeded.", func() {
+			Eventually(func() bool {
+				releaseCreated, err := framework.AsKubeAdmin.ReleaseController.GetFirstReleaseInNamespace(devNamespace)
+				if releaseCreated == nil || err != nil {
+					return false
+				}
+
+				return releaseCreated.HasStarted() && releaseCreated.IsDone() && releaseCreated.Status.Conditions[0].Status == "True"
+			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("tests that copying application and component works as designed.", func(ctx SpecContext) {
+			workingDir, err := os.Getwd()
+			if err != nil {
+				GinkgoWriter.Printf(err.Error())
+			}
+
+			// Download copy-applications.sh script from release-utils repo
+			args := []string{"https://raw.githubusercontent.com/hacbs-release/release-utils/main/copy-application.sh", "-o", "copy-application.sh"}
```

Comment:
@scoheb just wondering, how is this app/component copy part going to be handled in stage/production environment? will release-service be taking care of it?